common/test/model/ContentTest.scala

```diff
@@ -116,6 +116,28 @@ class ContentTest extends FlatSpec with Matchers with implicits.Dates {
     Content(sensitive).shouldHideAdverts should be(true)
   }
 
+  it should "detect if article requires membershipAccess" in {
+
+    conf.Switches.MembersAreaSwitch.switchOn()
```

Comment:
@robertberry I know you suggested it was OK to only check the existence of the classname but just to be on the safe side I added some tests for the `requiresMembershipAccess` flag.

Added `conf.Switches.MembersAreaSwitch.switchOn()` here too, but there wasn't another example of this in this file so not sure if that's needed?